doc/sphinx/source/changelog.rst

```diff
@@ -4,6 +4,103 @@ Changelog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Remove installation of R dependencies from the help message (`#2761 <https://github.com/ESMValGroup/ESMValTool/pull/2761>`__) `Rémi Kazeroni <https://github.com/remi-kazeroni>`__
+
+Bug fixes
+~~~~~~~~~
+
+-  Fix misplaced provenance records from IPCC AR5 Ch.12 diags (`#2758 <https://github.com/ESMValGroup/ESMValTool/pull/2758>`__) `Axel Lauer <https://github.com/axel-lauer>`__
+-  Fix `esmvaltool.utils.testing.regression.compare` module to run with Python<3.10 too (`#2778 <https://github.com/ESMValGroup/ESMValTool/pull/2778>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Fixed small bug that could lead to wrong pr units in `monitor/multi_datasets.py` (`#2788 <https://github.com/ESMValGroup/ESMValTool/pull/2788>`__) `Manuel Schlund <https://github.com/schlunma>`__
+-  Pin `xgboost>1.6.1` so we avert documentation failing to build with `1.6.1` (`#2780 <https://github.com/ESMValGroup/ESMValTool/pull/2780>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Fixed wrong latitudes in NDP CMORizer (`#2832 <https://github.com/ESMValGroup/ESMValTool/pull/2832>`__) `Manuel Schlund <https://github.com/schlunma>`__
+-  Fix indexer in Autoassess supermeans module use a tuple of `(slice(), idx, idx)` (`#2838 <https://github.com/ESMValGroup/ESMValTool/pull/2838>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Replace xarray ufuncs with bogstandard numpy in weighting/climwip/calibrate_sigmas.py (`#2848 <https://github.com/ESMValGroup/ESMValTool/pull/2848>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Fix units MERRA2 CMORizer (`#2850 <https://github.com/ESMValGroup/ESMValTool/pull/2850>`__) `Axel Lauer <https://github.com/axel-lauer>`__
+-  Fix bug when using log-scale y-axis for ocean transects. (`#2862 <https://github.com/ESMValGroup/ESMValTool/pull/2862>`__) `Tomas Torsvik <https://github.com/TomasTorsvik>`__
+
+Community
+~~~~~~~~~
+
+-  Add MO-paths to config file (`#2784 <https://github.com/ESMValGroup/ESMValTool/pull/2784>`__) `mo-tgeddes <https://github.com/mo-tgeddes>`__
+
+Deprecations
+~~~~~~~~~~~~
+
+-  Recipe `recipe_esacci_oc.yml` replace with new regrid scheme `nearest_extrapolate` (`#2841 <https://github.com/ESMValGroup/ESMValTool/pull/2841>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+
+Documentation
+~~~~~~~~~~~~~
+
+-  Update release schedule for v2.7 (`#2747 <https://github.com/ESMValGroup/ESMValTool/pull/2747>`__) `Bouwe Andela <https://github.com/bouweandela>`__
+-  Add Met Office installation method (`#2751 <https://github.com/ESMValGroup/ESMValTool/pull/2751>`__) `mo-tgeddes <https://github.com/mo-tgeddes>`__
+-  Add release dates for 2023 (`#2769 <https://github.com/ESMValGroup/ESMValTool/pull/2769>`__) `Rémi Kazeroni <https://github.com/remi-kazeroni>`__
+-  Made `maintainer` entry mandatory for published recipes (`#2703 <https://github.com/ESMValGroup/ESMValTool/pull/2703>`__) `Manuel Schlund <https://github.com/schlunma>`__
```

Comment:
I think this should have been flagged "backward incompatible changes". All the integrated recipes should continue to run with v2.7.0 since an "unmaintained" entry has been added to recipes without a maintainer. But I'm not so sure about "user recipes" (outside of the main branch). If a recipe developer had started working on a recipe based on the past release, chances are high that such recipe still does not contain a maintainer entry and will not be runnable with v2.7.0. In v2.6.0, 39 of the 125 recipes did not have a maintainer entry. Sorry for bringing that up so late in the release cycle. I think such changes should be handled in accordance with the agreed Backward compatibility policy (see #2879) and guidance should be provided to fix user recipes. What do you think @valeriupredoi @schlunma ?

Reply:
Hi @remi-kazeroni,

Great that you're already carefully looking for backwards incompatible changes! In this case, we added a unit test that all recipes in the ESMValTool repository contain a maintainer https://github.com/ESMValGroup/ESMValTool/issues/2675. We discussed this at the workshop in spring, and back then we thought that this would qualify as backwards compatible, because it does not affect any of the users of ESMValTool recipes, nor does it affect any recipes developed outside the ESMValTool repository.

Reply:
Thanks for clarifying @bouweandela. Sorry, I totally misremembered about this PR and had the impression that the maintainer entry would be needed to run a recipe with v2.7.0. This is actually not the case so I also think this is backwards compatible 👍